packages/devextreme/testing/testcafe/tests/editors/dateRangeBox/calendar.ts

```diff
@@ -696,6 +696,39 @@ test('Disabled dates on inputs focus (disableOutOfRangeSelection: true)', async
   }, '#dateRangeBox');
 });
 
+test('Hover styles appearance on second date pick (disableOutOfRangeSelection=true)', async (t) => {
```

Comment:
shouldn't we just check that class is added instead of making a screenshot?

Reply:
great idea! will do